specs/advanced-data-layouts/hamt/fixture/alice-words/index.md

```diff
@@ -13,8 +13,8 @@ This fixture builds a HAMT cataloging the locations of all words appearing in th
 * Words are matched using a simple `/\w+/` regular expression.
 * The HAMT uses a `bitWidth` of `5` (maximum of 32 entries or buckets per node) and a `bucketSize` of `3`.
 * Blocks are encoded using DAG-CBOR and CIDs use a SHA2-256 multihash.
-* Each entry in keyed by the word (as `Bytes` as per the HashMap spec) and the values are stored inline.
-* Values are an array of the line and column location of each occurance of the word, in order of appearance.
+* Each entry is keyed by the word (as `Bytes` as per the [HashMap spec](../../spec.md)) and the values are stored inline.
```

Comment:
```suggestion
* Each entry is keyed by the word (as `Bytes` as per the [HashMap spec](../../spec/)) and the values are stored inline.
```

build is failing because we have to do this stupid dance on the compiled version; off the top of my head I think this is the correct version of the link.

you can also change "HashMap" to "HAMT" if you like since it's already been changed all over this repo